packages/web/src/scss/components/CheckboxField/_CheckboxField.scss

```diff
@@ -1,89 +1,77 @@
 @use 'theme';
+@use '../../theme/form-fields' as form-fields-theme;
 @use '../../tools/accessibility';
```

Comment:
Please make sure all Sass modules (in all files) are actually called. Many of them have become unused after the abstraction of form-field mixins.